src/nvim/lua/executor.c

```diff
@@ -132,6 +167,16 @@ static int nlua_state_init(lua_State *const lstate) FUNC_ATTR_NONNULL_ALL
   lua_setfield(lstate, -2, "stricmp");
 
   lua_setglobal(lstate, "vim");
+
+  // expose main_loop's UV loop for luv
+  lua_pushstring(lstate, "uv_loop");
+  lua_pushlightuserdata(lstate, &main_loop.uv);
+  lua_rawset(lstate, LUA_REGISTRYINDEX);
+
+  // schedule
+  lua_pushcfunction(lstate, &nlua_schedule);
+  lua_setglobal(lstate, "nvim_schedule");
```

Comment:
Rather set `schedule` on the vim object, compare with the addition of `vim.stricmp` above.

Reply:
Makes sense! Done.